### ISSUE TITLE
Accessibility: Table columns can not be sorted using keyboard only

### ISSUE THREAD

**Issue by sedlatschek** [laravel/nova-issues#3427]
- Laravel Version: 8.44.0
- Nova Version: 3.26.1
- PHP Version: 7.4.6
- Database Driver & Version: MySQL 5.7
- Operating System and Version: Win10
- Browser type and version: Chrome 88

### Description:
Table columns in index views can not be reached using the keyboard only. Therefor blind users are unable to change the sorting of the table. Please use buttons instead of spans and add an `aria-sort` attribute to indicate the sort direction.

![image](https://user-images.githubusercontent.com/58665536/121049488-0400c000-c7b8-11eb-9529-11cf8f6fdfd9.png)

Laravel Nova is not accessible. This is a problem in enterprise environments, where accessibility is a must requirement.

**Comment by davidhemphill** [comment]
👋 Hi there! In order to keep this repository focused on bug reports, we auto-close feature requests and requests for help. Feel free to post your feature requests so others can discuss and add reactions. We'll keep an eye on them for later planning.
